addons/crm/migrations/9.0.1.0/pre-migration.py

```diff
@@ -33,6 +40,16 @@
     ],
 }
 
+xmlid_renames = [
```

Comment:
This should be done on crm_phonecall itself, not here. Result: if you don't have crm_phonecall, XML-IDs are removed at the end. If you have it, it doesn't crush.